code/game/objects/effects/contraband.dm

```diff
@@ -603,7 +603,7 @@
 
 /obj/structure/sign/poster/official/no_erp
 	name = "No ERP"
-	desc = "This poster reminds the crew that Eroticism, Rape and Pornography are banned on Nanotrasen stations."
+	desc = "This poster reminds the crew that Eroticism, Rubbing and Pornography are banned on Nanotrasen stations."
```

Comment:
```suggestion
	desc = "This poster reminds the crew that Eroticism, Rape and Pornography are banned on Nanotrasen stations."
```

Reply:
rubbing is a weird word, could just have no word there and have the R represent the second letter of eroticism